gradle.properties

```diff
@@ -18,4 +18,8 @@
 # org.gradle.parallel=true
 android.enableJetifier=true
 android.useAndroidX=true
-kapt.incremental.apt=true
\ No newline at end of file
+kapt.incremental.apt=true
+
+# When building in TeamCity, the build was failing on jetifier.  This worked around it.
+# https://github.com/square/moshi/issues/804#issuecomment-466456323
+android.jetifier.blacklist = kotlin-compiler-embeddable-.*\\.jar
```

Comment:
@bryanwussow I had to change one aspect of the build (jetifier setting).  Could you please test out the resulting build?